modules/services/polybar.nix

```
@@ -205,7 +205,7 @@ in {
 
       Service = {
         Type = "forking";
-        Environment = "PATH=${cfg.package}/bin:/run/wrappers/bin";
+        Environment = "PATH=$PATH:${cfg.package}/bin:/run/wrappers/bin";
```

```suggestion
        Environment = "PATH=$PATH\${PATH:+:}${cfg.package}/bin:/run/wrappers/bin";
```

I don't think it's even possible for the PATH to be empty in the service environment but this copies the logic from nixpkgs' `makeWrapper`'s `--prefix`

---

Done, thank you :+1: